sources/au.json

```diff
@@ -42,5 +42,21 @@
             "bryce@esonica.com"
         ],
         "active": true
+    },
+    {
+        "url": "http://www.epa.sa.gov.au/air_quality/pi.txt",
+        "adapter": "au_sa",
+        "name": "Australia - South Australia",
+        "country": "AU",
+        "resolution": "1 hr",
+        "sourceURL": "https://data.sa.gov.au/data/dataset/recent-air-quality",
+        "attribution": [{
```

Comment:
No worries. I'll might try to follow up later with changes to the CONTRIBUTING docs because this was a major source of confusion for me.

Reply:
Right so should the source.name be consistent with other sources ie "Australia - South Australia" or should it be the name of the entity who produce the source "Environment Protection Authority (EPA), South Australia".

I think the first is best, considering there are probably a number of different "EPA"'s globally.

Reply:
What you did looks correct!

Reply:
> No worries. I'll might try to follow up later with changes to the CONTRIBUTING docs because this was a major source of confusion for me.

Agreed it is confusing and unclear. The CONTRIBUTING docs are very much in need of an update, so would really appreciate any changes you may have.